app/config.py

```diff
@@ -519,6 +519,8 @@ class Config(object):
         "job": "{}-dvla-file-per-job".format(os.getenv("NOTIFY_ENVIRONMENT", "development")),
         "notification": "{}-dvla-letter-api-files".format(os.getenv("NOTIFY_ENVIRONMENT", "development")),
     }
+    SERVICE_ANNUAL_EMAIL_LIMIT = env.int("SERVICE_ANNUAL_EMAIL_LIMIT", 10_000_000)
+    SERVICE_ANNUAL_SMS_LIMIT = env.int("SERVICE_ANNUAL_SMS_LIMIT", 25_000)
```

Comment:
Where do we define the daily limits? I expected to see them somewhere here too...

Reply:
We hard-coded defaults in the [front end config](https://github.com/cds-snc/notification-admin/blob/84e6be282c84c1b4d00972042f05d29432e13185/app/config.py#L92) for display in `service-settings`. I also expected to see this configured in API but did not.